roadmap/implementers-guide/src/types/network.md

```diff
@@ -142,6 +142,23 @@ enum CollationProtocolV1 {
 These updates are posted from the [Network Bridge Subsystem](../node/utility/network-bridge.md) to other subsystems based on registered listeners.
 
 ```rust
+struct NewGossipTopology {
+	/// The session index this topology corresponds to.
+	session: SessionIndex,
+	/// Neighbors in the 'X' dimension of the grid.
+	our_neighbors_x: HashMap<AuthorityDiscoveryId, TopologyPeerInfo>,
+	/// Neighbors in the 'Y' dimension of the grid.
+	our_neighbors_y: HashMap<AuthorityDiscoveryId, TopologyPeerInfo>,
+}
+
+struct TopologyPeerInfo {
+	/// The validator's known peer IDs.
+	peer_ids: Vec<PeerId>,
+	/// The index of the validator in the discovery keys of the corresponding
+	/// `SessionInfo`.
+	validator_index: ValidatorIndex,
```

Comment:
It's might be worth pointing out in the docs that usually `ValidatorIndex` is limited to parachain validators, but that's not the case here, although the first `session_info.validators` indices should match. This is documented in `SessionInfo`, but not clear from using the same type for indexing type. Or, alternatively, using a separate type.